### PR TITLE
My suggestion changes.

### PR DESCRIPTION
**Pull request by hosseinseilani**
Hi. my suggestion changes.

thanks a lot

### CODE REVIEW

**Comment by pieroproietti** [comment]
Sorry, I must refuse, becouse I discovered it after I created my changes.

Note: actually we have /addons/template  with isolinux.main.cfg and grub.main.cfg (the old isolinux.template and grub.template), and on `/addons/eggs/theme/livecd/` we have: `isolinux.theme.cfg` and `grub.theme.cfg`.

In the next commit I will put all under `/addons/eggs/theme/livecd/`.

So, in your theme too, we will have - under the same path `/vendors/predator/theme/livecd` - on wardrobe: 

In short, adapt your theme with this two template for isolinux and grub:
* isolinux.main.cfg
* isolinux.theme.cfg
* grub.main.cfg
* grub.theme.cfg

Main is for menues, themes are for colors, fonts and so on. 

Thanks Husein
